forloop_modules/utils/various.py

```diff
@@ -0,0 +1,13 @@
+import inspect
+
+from forloop_modules.utils.definitions import JSON_SERIALIZABLE_TYPES, REDIS_STORED_TYPES
+
+
+def is_value_serializable(value) -> bool:
+    return True if type(value) in JSON_SERIALIZABLE_TYPES else False
+
+
+def is_value_redis_compatible(value) -> bool:
+    return (
+        type(value) in REDIS_STORED_TYPES or inspect.isfunction(value) or inspect.isclass(value)
+    )
```

Comment:
```suggestion
    is_value_a_callable = inspect.isfunction(value) or inspect.isclass(value)
    is_value_redis_compatible = type(value) in REDIS_STORED_TYPES or is_value_a_callable
    return is_value_redis_compatible
```

Reply:
These last 2 comments are subjective, functionally wise it's perfectly fine 😄

Reply:
Great code can be read like a book :)